about.html

```diff
@@ -0,0 +1,113 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="https://use.typekit.net/mxx0rob.css">
+  <link rel="stylesheet" href="style.css">
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.1/dist/css/bootstrap.min.css" rel="stylesheet"
+    integrity="sha384-iYQeCzEYFbKjA/T2uDLTpkwGzCiq6soy8tYaI1GyVh/UjpbCx/TYkiZhlZB6+fzT" crossorigin="anonymous">
+  <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.1/dist/js/bootstrap.bundle.min.js"
+    integrity="sha384-u1OknCvxWvY5kfmNBILK2hRnQC3Pr17a+RTT6rIHI7NnikvbZlHgTPOOmMi466C8"
+    crossorigin="anonymous"></script>
+  <script src="https://kit.fontawesome.com/bf66e12569.js" crossorigin="anonymous"></script>
+  <title>About</title>
+</head>
+
+<body>
+  <header class="header">
+    <div id="hide-div"><i class="fa-solid fa-bars menu-bars"></i></div>
+    <div class="mobile-menu">
+      <i class="fa-solid fa-x menu-x"></i>
+      <ul class="menu-mobile-list unic-font">
+        <li class="menu-mobile-list-itens"><a href="about.html" class="no-underscore color-three">About</a></li>
+        <hr class="menu-mobile-line">
+        <li class="menu-mobile-list-itens color-three"><a href="index.html" class="no-underscore color-three">Program</a></li>
+        <hr class="menu-mobile-line">
+        <li class="menu-mobile-list-itens color-three">Join</li>
+        <hr class="menu-mobile-line">
+        <li class="menu-mobile-list-itens color-three">Sponsor</li>
+        <hr class="menu-mobile-line">
+        <li class="menu-mobile-list-itens color-three">News</li>
+      </ul>
```

Comment:
- [ ] you have the logo on left of your page please inline with the text below as provided in the design. See image below for reference
### your design
![his](https://user-images.githubusercontent.com/81727988/191886315-2f81350b-7ea4-4d88-bbc3-898eae073033.png)

### Figma design
![figma](https://user-images.githubusercontent.com/81727988/191886155-78685d42-0c00-441e-a587-0c47a7c4c637.png)